prod-cockpit-rcar.yaml

```diff
@@ -135,11 +135,11 @@ components:
 
         # get xen customized under cr7
         - [XEN_URL_pn-xen-tools, "%{XEN_URL}"]
-        - [SRCREV_pn-xen-tools, "d9703fe08c3d32dbfb89ab10502ad2ffeb52de62"]
+        - [SRCREV_pn-xen-tools, "995f0fcf697d8d8690ed02e0990b2be074ebe8cb"]
```

Comment:
Maybe it is time to introduce a single variable to hold revision? Like `XEN_URL` above.

Reply:
Great idea!